examples/sensor-data-scala/build.sbt

```diff
@@ -3,8 +3,9 @@ import sbt._
 import sbt.Keys._
 
 lazy val sensorData =  (project in file("."))
-    .enablePlugins(CloudflowAkkaStreamsApplicationPlugin)
+    .enablePlugins(CloudflowAkkaStreamsApplicationPlugin, ScalafmtPlugin)
     .settings(
+      scalafmtOnCompile := true,
```

Comment:
[minor] this might have a short-term impact on the docs since the project setup is used as a snippet but @skyluc is already working on decoupling the examples from the snippets used in the docs so I think it is ok.